### DISCUSSION TITLE
Any discount for NPOs?

### DISCUSSION THREAD

**Discussion by pkiula** [plausible/analytics#1872] [Anything Else]
Hello. I'd love to use this beautiful, simple, elegant looking analytics for my clients. I can just passthru the costs to clients as they can also see the value of this simple insights platform. But some of them are NPOs and operate on tight budgets. Would it be possible to allow them to use this for free? We can share 501(c)3 certification etc as proof.

**Comment by metmarkosaric** [comment]
thanks for reaching out @pkiula and for the kind words! we don't have any free plans unfortunately. we're 100% self-funded and bootstrapped and we don't want to make money selling your data. it takes a lot of time to develop and maintain Plausible and subscription fees is what helps us keep it all sustainable. while we don't have free plans, we're happy to offer registered NPOs a discount. please do [reach out](https://plausible.io/contact)
